index.html

```diff
@@ -2097,6 +2095,19 @@ <h4>Semantic Interoperability</h4>
 <code>@protected</code> feature in the JSON-LD 1.1 specification.
             </li>
           </ul>
+          <p>
+A human-readable document describing the expected order of values for the
+<code>@context</code> <a>property</a> SHOULD be available. A machine-readable
+description (i.e. a normal JSON-LD "context" document) SHOULD be available
```

Comment:
A JSON-LD Context isn't a SHOULD, it's a MUST, as described elsewhere in the specification (so we shouldn't restate it here).

Reply:
Fixed in https://github.com/w3c/vc-data-model/pull/546/commits/0221fbb2eb2833505848649e65980e38110d0f55 ... but using your text.